jib-gradle-plugin/build.gradle

```diff
@@ -23,8 +23,7 @@ dependencies {
   testImplementation "org.slf4j:slf4j-api:${dependencyVersions.SLF4J_API}"
   testImplementation "com.github.stefanbirkner:system-rules:${dependencyVersions.SYSTEM_RULES}"
 
-  testImplementation project(path:':jib-plugins-common', configuration:'tests')
-  integrationTestImplementation project(path:':jib-core', configuration:'integrationTests')
+  testImplementation project(path:':jib-testing-common')
```

Comment:
nit: for consistency, `testImplementation project(':jib-testing-common')` (without specifying the `path` argument name).

And does the order of `testImplementation` matter? In `jib-core`, it's the first entry.